demisto_sdk/commands/validate/validators/AS_validators/AS_100_aggregated_script_has_tpb.py

```diff
@@ -0,0 +1,50 @@
+from typing import ClassVar, Final, Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+NO_TESTS_FORMAT: Final[list[str]] = ["No tests (auto formatted)"]
+MISSING_TPB_MESSAGE: Final[str] = "Script {name} is missing a TPB"
+AGGREGATED_SCRIPTS_PACK_NAME: Final[str] = "Aggregated Scripts"
+
```

Comment:
```suggestion
ContentTypes = Script
```
this way it will only run validation on scripts